src/main/java/racingcar/controller/Controller.java

```diff
@@ -0,0 +1,36 @@
+package racingcar.controller;
+
+import racingcar.model.Cars;
+import racingcar.model.Race;
+import racingcar.view.Input;
+import racingcar.view.Output;
+
+public class Controller {
+    public void startGame() {
+        Input input = new Input();
+        String carNames = input.requestCarNames();
+        String raceRounds = input.requestRaceRounds();
```

Comment:
Input 클래스가 따로 상태를 관리하지 않고 단순히 입력값에 대한 검증을 수행하는 거라면 static 클래스로 선언해서 사용해도 괜찮지 않을까요?!

```suggestion
        String carNames = Input.requestCarNames();
        String raceRounds = Input.requestRaceRounds();
```

Reply:
아 넵 그렇군요! 그렇게 하는 것이 더 좋겠네요!